content/doc/advanced/hacking-on-gitea.zh-cn.md

```diff
@@ -15,7 +15,7 @@ menu:
 
 # Hacking on Gitea
 
-首先你需要一些运行环境，这和 [从源代码安装](/zh-cn/install-from-source) 相同，如果你还没有设置好，可以先阅读那个章节。
+首先你需要一些运行环境，这和 [从源代码安装]({{< relref "from-source.zh-cn.md" >}}) 相同，如果你还没有设置好，可以先阅读那个章节。
```

Comment:
as from-source.zh-cn.md is unique in this project, `{{< relref "from-source.zh-cn.md" >}}` would point to `content/doc/installation/from-source.zh-cn.md`, and this fix with the expected result in my local workspace, `{{< relref "../installation/from-source.zh-cn.md" >}}` is incorrect, i think we can use `({{< relref "doc/installation/from-source.zh-cn.md" >}})`(tested normally in my local).